src/test/java/ai/elimu/LetterToAllophoneMappingsControllerTest.java

```diff
@@ -31,57 +30,54 @@
 import static org.hamcrest.CoreMatchers.*;
 import static org.junit.Assert.assertThat;
 
-@RunWith(MockitoJUnitRunner.class)
-public class LetterToAllophoneMappingsControllerTest {
-
-    @Mock
+public class LetterToAllophoneMappingsControllerTest extends RestControllerTest{
+    @Autowired
     LetterToAllophoneMappingDao letterToAllophoneMappingDao;
 
-    @InjectMocks
-    LetterToAllophoneMappingsController letterToAllophoneMappingsController = new LetterToAllophoneMappingsController();
+    @Autowired
+    private AllophoneDao allophoneDao;
 
     @Autowired
+    private LetterDao letterDao;
+
+    @Autowired
+    LetterToAllophoneMappingsController letterToAllophoneMappingsController;
+
     private MockMvc mockMvc;
 
     private Logger logger = LogManager.getLogger();
 
     @Before
-    public void setup() throws JsonProcessingException {
-        JSONObject tesJson = new JSONObject("{\n" +
-                "    \"allophones\": [\n" +
-                "        {\n" +
-                "            \"revisionNumber\": 2,\n" +
-                "            \"diacritic\": false,\n" +
-                "            \"valueIpa\": \"æ\",\n" +
-                "            \"id\": 5,\n" +
-                "            \"soundType\": \"VOWEL\",\n" +
-                "            \"usageCount\": 770\n" +
-                "        }\n" +
-                "    ],\n" +
-                "    \"id\": 2,\n" +
-                "    \"letters\": [\n" +
-                "        {\n" +
-                "            \"revisionNumber\": 18,\n" +
-                "            \"diacritic\": false,\n" +
-                "            \"text\": \"a\",\n" +
-                "            \"id\": 1,\n" +
-                "            \"usageCount\": 914\n" +
-                "        }\n" +
-                "    ],\n" +
-                "    \"usageCount\": 77\n" +
-                "}");
-        ObjectMapper mapper = new ObjectMapper();
-        LetterToAllophoneMapping letterToAllophoneMapping =
-                mapper.readValue(tesJson.toString(), LetterToAllophoneMapping.class);
-
-        List<LetterToAllophoneMapping> testList = new ArrayList<>();
-        testList.add(letterToAllophoneMapping);
-        Mockito.when(letterToAllophoneMappingDao.readAllOrderedByUsage()).thenReturn(testList);
+    public void setup() throws JsonProcessingException {Allophone allophone = new Allophone();
+        allophone.setValueIpa("ɛɛ");
```

Comment:
If you would like to check some examples of existing Letter-to-Allophone mappings, see http://eng.elimu.ai/content/letter-to-allophone-mapping/list